pkgs/tools/security/sha1collisiondetection/default.nix

```diff
@@ -0,0 +1,39 @@
+{ stdenv, fetchFromGitHub, libtool }:
+
+stdenv.mkDerivation  rec {
+  pname = "sha1collisiondetection";
+  version = "git-20170221";
```

Comment:
@grahamc : would the Security Team consider the above proposal to replace the old sha1sum with the new one that is able to detect collision attacks, and deprecate the 'shattered' one?